Formula/libeatmydata.rb

```diff
@@ -0,0 +1,96 @@
+require "erb"
+
+class Libeatmydata < Formula
+  desc "library to (transparently) disable fsync and friends."
+  homepage "https://www.flamingspork.com/projects/libeatmydata/"
+  url "https://www.flamingspork.com/projects/libeatmydata/libeatmydata-105.tar.gz"
+  sha256 "bdd2d068b6b27cf47cd22aa4c5da43b3d4a05944cfe0ad1b0d843d360ed3a8dd"
+
+  depends_on "coreutils"
+
+  # This patch fixes three things:
+  #
+  # 1) `dpkg-architecture` only exists on Debian based platforms. This
+  #    doesn't make much sense on OS X, so we'll just take it out.
+  #
+  # 2) OS X's built-in `readlink` doesn't recognize the `-f`
+  #    (`--canonicalize`) flag, which is a GNU extension. The
+  #    "coreutils" formula isn't linked in, so we have to modify this
+  #    to use `greadlink` explicitly.
+  #
+  # 3) OS X doesn't seem to define an `open64` function by default. A
+  #    more thorough fix would be to ensure it's covered in the
+  #    Automake script, but I have no idea how to do that.
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission in the formula file. We'll need upstream to agree to accept OS X patches before we can include this. Thanks!